carlos/rocker/css/style.css

```diff
@@ -0,0 +1,272 @@
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
+    /** 1rem = 10px **/
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: inherit;
+}
+
+body {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1.6rem;
```

Comment:
Not only you should avoid using relative values for font size. You should stick to the same value type in your project and not have sometimes percentages, sometimes rems, sometimes pxs....